app.js

```diff
@@ -13,9 +13,13 @@ const dirname = path.resolve();
 
 const app = express();
 const swaggerDoc = swaggerYaml.load('./swagger.yaml');
+const tutorialWinston = require("./config/tutorial.winston");
 
-// eslint-disable-next-line no-console
-promise.then(() => console.log("Connection successful!"));
+if (promise) {
```

Comment:
This is not how a promise works. Please read this docs:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise

CC: @ashishkava @jaimin45